client/flutter/lib/module_account/ui/screen/presonal_data_screen.dart

```diff
@@ -0,0 +1,231 @@
+                  ),
+                ),
+              ),
+            ),
+          ),
+          elevation: 0,
+        ),
+        body:Stack(
+          children: [
+            ListView(
+              physics: BouncingScrollPhysics(
+                  parent: AlwaysScrollableScrollPhysics()),
+              children: [
+                SizedBox(
+                  height: 16,
+                ),
+                Container(
+                  height: 150,
+                  child: Center(
+                    child: SizedBox(
+                      width: 150,
+                      height: 150,
+                      child: Stack(
+                        children: [
+                          ClipRRect(
+                            borderRadius: BorderRadius.circular(18),
+                            child: Image.network(
+                              'https://img.bundesliga.com/tachyon/sites/2/2021/06/kimmich-look-fcb-bayern.jpg?crop=128px,0px,1351px,1080px',
+                              fit: BoxFit.cover,
+                              width: 150,
+                              height: 150,
+                            ),
+                          ),
+                          Padding(
+                            padding: const EdgeInsets.all(8.0),
+                            child: Align(
+                              alignment: AlignmentDirectional.bottomEnd,
+                              child: Opacity(
+                                opacity: 0.9,
+                                child: Container(
+                                  decoration: BoxDecoration(
+                                      borderRadius: BorderRadius.circular(18),
+                                      color: Theme.of(context).primaryColor),
+                                  child: Padding(
+                                    padding: const EdgeInsets.all(8.0),
+                                    child: Icon(
+                                      Icons.add_a_photo_rounded,
+                                      color: Colors.white,
+                                    ),
+                                  ),
+                                ),
+                              ),
+                            ),
+                          )
+                        ],
+                      ),
+                    ),
+                  ),
+                ),
+                Form(
+                  key: _personal_data,
+                  child: Padding(
+                    padding: const EdgeInsets.only(right: 16.0, left: 16),
+                    child: Flex(
+                      direction: Axis.vertical,
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        ListTile(
+                          title: Padding(
+                            padding: const EdgeInsets.all(8.0),
+                            child: Text(
+                              S.of(context).name,
```

Comment:
what 'S' refer to?

Reply:
messages generated class called S we used it to access translated messages